--- a/misc/create_inode.c	2018-12-06 00:48:56.425304202 -0800
+++ b/misc/create_inode.c	2018-12-06 00:43:40.189082525 -0800
@@ -15,6 +15,7 @@
 
 #include "config.h"
 #include <time.h>
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <unistd.h>
 #include <limits.h> /* for PATH_MAX */
--- a/lib/ext2fs/ismounted.c	2018-12-06 00:48:47.749490237 -0800
+++ b/lib/ext2fs/ismounted.c	2018-12-06 00:41:29.550880983 -0800
@@ -47,6 +47,7 @@
 #include <string.h>
 #include <sys/stat.h>
 #if HAVE_SYS_TYPES_H
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 #endif
 
--- ./debugfs/debugfs.c.orig	2018-12-08 22:00:01.442069224 -0800
+++ ./debugfs/debugfs.c	2018-12-08 22:01:09.455585020 -0800
@@ -15,6 +15,7 @@
 #include <ctype.h>
 #include <string.h>
 #include <time.h>
+#include <sys/sysmacros.h>
 #include <libgen.h>
 #ifdef HAVE_GETOPT_H
 #include <getopt.h>
